templates/taxi/car_detail.html

```diff
@@ -3,14 +3,18 @@
 {% block content %}
   <h1>
     {{ car.model }}
-    <a href="{% url 'taxi:car-delete' pk=car.id %}" class="btn btn-danger link-to-page">
-      Delete
-    </a>
-
-    <a href="{% url 'taxi:car-update' pk=car.id %}" class="btn btn-secondary link-to-page">
-      Update
-    </a>
   </h1>
+  {% if request.user in car.drivers.all%}
```

Comment:
There is a missing space before the `%}` in `{% if request.user in car.drivers.all%}`. It should be `{% if request.user in car.drivers.all %}` for consistency and readability.

Reply:
done